guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -7,8 +7,8 @@ After your upgrade is complete, you can then decommission the earlier version of
 
 Use the following procedures to clone your {Project} instances to preserve your environments in preparation for upgrade.
 
-The {Project} clone tool does not support migrating a {SmartProxyServer} to {EL} 7.
-Instead you must backup the existing {SmartProxyServer}, restore it on {EL} 7, then reconfigure {SmartProxyServer}.
+The {Project} clone tool does not support migrating a {SmartProxyServer} to {EL} 8.
+Instead you must backup the existing {SmartProxyServer}, restore it on {EL} 8, then reconfigure {SmartProxyServer}.
```

Comment:
I do not know whether the procedure in this file is the same for EL 8. Is there someone who could confirm this?

Reply:
"Cloning to the Target Server", step 4. should probably enable RHEL8 packages. Otherwise I wonder if all stays the same, I'm trying to check.

Reply:
Thanks so much for checking! I'll ask in the satellite IRC channel, too